ibm/resource_ibm_is_virtual_endpoint_gateway.go

```diff
@@ -162,9 +162,10 @@ func resourceIBMISEndpointGateway() *schema.Resource {
 							Description: "The target crn",
 						},
 						isVirtualEndpointGatewayTargetResourceType: {
-							Type:        schema.TypeString,
-							Required:    true,
-							Description: "The target resource type",
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validateAllowedStringValue([]string{"provider_cloud_service", "provider_infrastructure_service"}),
```

Comment:
Can you please use InvokeValidator instead of validateAllowedStringValue

Reply:
made the required changes and tested as well
<img width="715" alt="Screenshot 2021-09-01 at 4 35 58 PM" src="https://user-images.githubusercontent.com/78336632/131661000-d3d3a423-9489-4cb0-9a93-e9e7a23723d0.png">